snmp/datadog_checks/snmp/data/default_profiles/dell-emc-data-domain.yaml

```
@@ -0,0 +1,336 @@
+  - 1.3.6.1.4.1.19746.3.1.6    # EMC dd400g
+  - 1.3.6.1.4.1.19746.3.1.7    # EMC dd460g
+  - 1.3.6.1.4.1.19746.3.1.8    # EMC dd560
+  - 1.3.6.1.4.1.19746.3.1.9    # EMC dd560g
+  - 1.3.6.1.4.1.19746.3.1.10    # EMC dd580
+  - 1.3.6.1.4.1.19746.3.1.11    # EMC dd580g
+  - 1.3.6.1.4.1.19746.3.1.12    # EMC dd565
+  - 1.3.6.1.4.1.19746.3.1.13    # EMC dd530
+  - 1.3.6.1.4.1.19746.3.1.14    # EMC dd510
+  - 1.3.6.1.4.1.19746.3.1.15    # EMC dd120
+  - 1.3.6.1.4.1.19746.3.1.16    # EMC dd690
+  - 1.3.6.1.4.1.19746.3.1.17    # EMC dd690g
+  - 1.3.6.1.4.1.19746.3.1.18    # EMC dd660
+  - 1.3.6.1.4.1.19746.3.1.19    # EMC dd880
+  - 1.3.6.1.4.1.19746.3.1.20    # EMC dd880g
+  - 1.3.6.1.4.1.19746.3.1.21    # EMC dd610
+  - 1.3.6.1.4.1.19746.3.1.22    # EMC dd630
+  - 1.3.6.1.4.1.19746.3.1.23    # EMC dd140
+  - 1.3.6.1.4.1.19746.3.1.24    # EMC dd670
+  - 1.3.6.1.4.1.19746.3.1.25    # EMC dd860
+  - 1.3.6.1.4.1.19746.3.1.26    # EMC dd860g
+  - 1.3.6.1.4.1.19746.3.1.27    # EMC dd890
+  - 1.3.6.1.4.1.19746.3.1.28    # EMC dd640
+  - 1.3.6.1.4.1.19746.3.1.29    # EMC dd620
+  - 1.3.6.1.4.1.19746.3.1.30    # EMC dd160
+  - 1.3.6.1.4.1.19746.3.1.31    # EMC ddintrepid
+  - 1.3.6.1.4.1.19746.3.1.32    # EMC dd4500
+  - 1.3.6.1.4.1.19746.3.1.33    # EMC dd7200
+  - 1.3.6.1.4.1.19746.3.1.34    # EMC ddve
+  - 1.3.6.1.4.1.19746.3.1.35    # EMC dd990
+  - 1.3.6.1.4.1.19746.3.1.36    # EMC dd2500
+  - 1.3.6.1.4.1.19746.3.1.37    # EMC dd4200
+
+metrics:
+  - MIB: HOST-RESOURCES-MIB
+    symbol:
+      name: memory.total
+      OID: 1.3.6.1.2.1.25.2.3.1.5.1
+  - MIB: HOST-RESOURCES-MIB
+    symbol:
+      name: memory.used
+      OID: 1.3.6.1.2.1.25.2.3.1.6.1
+  - MIB: HOST-RESOURCES-MIB
+    symbol:
+      OID: 1.3.6.1.2.1.25.2.3.1.6.6  # hrStorageUsed.6 = Memory buffers
+      name: hrMemoryBuffers
+  - MIB: HOST-RESOURCES-MIB
+    symbol:
+      OID: 1.3.6.1.2.1.25.2.3.1.6.7  # hrStorageUsed.7 = Cached memory
+      name: hrCachedMemory
+#  - MIB: DATA-DOMAIN-MIB
+#    symbol:
+#      name: fileSystemStatus
+#      OID: 1.3.6.1.4.1.19746.1.3.1.1.0
+#      enum:
+#        enabled: 1
+#        disabled: 2
+#        running: 3
+#        unknown: 4
+#        error: 5
+#        cleaning: 6
+#      TODO: enum in scalar metric is not supported yet (keep this metric and this comment in profile until it's fixed)
+#  - MIB: DATA-DOMAIN-MIB
+#    symbol:
+#      name: fileSystemVirtualSpace
+#      OID: 1.3.6.1.4.1.19746.1.3.1.2.0
+# TODO: octet string cannot be collected
+#  - MIB: DATA-DOMAIN-MIB
+#    symbol:
+#      name: fileSystemUpTime
+#      OID: 1.3.6.1.4.1.19746.1.3.1.3.0
+#      format: TODO_handle_conversion_to_one
+#  - MIB: DATA-DOMAIN-MIB
+#    symbol:
+#      name: fileSystemStatusMessage
+#      OID: 1.3.6.1.4.1.19746.1.3.1.4.0
+#      format: TODO_handle_conversion_to_one
+  - MIB: DATA-DOMAIN-MIB
+    table:
+      name: powerModuleTable
+      OID: 1.3.6.1.4.1.19746.1.1.1.1.1
+    symbols:
+      - name: dell.data_domain.powerModule
```

```suggestion
      - name: datadomain.powerModule
```

For this case, I think it could make sense to only use `datadomain` as prefix.

It looks like Data Domain has been acquired by Dell / EMC in the past https://www.dell.com/en-us/dt/corporate/newsroom/announcements/2009/07/20090708-02.htm

The rational is that if someday, data domain is acquired again by another company, using only `datadomain` as prefix will make it easier to migrate the profile.

Similar to other cases, it preferred to use prefix tied to the MIB. In this case, using prefix `datadomain` that is related to the mib name `DATA-DOMAIN-MIB`. It's not always the best strategy, but for this case, I think it can make sense.

---

makes sense, thank you